frontend/src/landing/menu/CustomGalleryColours.css

```diff
@@ -0,0 +1,19 @@
+button.image-gallery-icon:hover {
```

Comment:
Had to make a new file that didn't have *.module.css at the end as react renames the classes in those files.